rust/kcl-lib/std/prelude.kcl

```diff
@@ -536,3 +536,7 @@ export fn assert(
   /// If the value was false, the program will terminate with this error message
   error?: string,
 ) {}
+
+/// Set a label.
+@(impl = std_rust)
```

Comment:
Please mark this as experimental until we settle on the final API.
```suggestion
@(impl = std_rust, experimental = true)
```

You'll want to use `@settings(experimentalFeatures = allow)` at the top of your KCL file to be able to call it.